server/data/quotes.json

```diff
@@ -1,314 +1,318 @@
-    "author": "Gaston Bachelard"
- },
- {
-    "sentence": "One should not aim at being possible to understand but at being impossible to misunderstand.",
-    "author": "Marcus Fabius Quintilian"
- },
- {
-    "sentence": "With languages, you are at home anywhere.",
-    "author": "Edmund De Waal"
- },
- {
-    "sentence": "Maybe then you comprehend, speaking one language only is a prison.",
-    "author": "David Mitchell"
- },
- {
-    "sentence": "Language reflects the soul of people.",
-    "author": "Paul Schibler"
- },
- {
-    "sentence": "You are as many a person as languages you know.",
-    "author": "Armenian Proverb"
- },
- {
-   "sentence": "If culture was a house then language was the key to the front door, to all the rooms inside.",
-   "author": "Khaled Housseini"
- },
- {
-   "sentence": "Language is power, life and the instrument of culture, the instrument of domination and liberation.",
-   "author": "Angela Carter"
- },
- {
-    "sentence": "You want the 3 golden rules to learn any language? Use it or lose it. Love it or leave it. Feel it or fail it.",
-    "author": "Safir Kassim Boudjelal"
- },
- {
-   "sentence": "Learn everything you can, anytime you can, from anyone you can; there will always come a time when you will be grateful you did.",
-   "author": "Sarah Caldwell"
- },
- {"sentence": "Language is courage: the ability to conceive a thought, to speak it, and by doing so to make it true.",
-  "author": "Salman Rushdie"
- },
- {
-  "sentence": "We die. That may be the meaning of life. But we do language. That may be the measure of our lives.",
-  "author": "Toni Morrison"
- },
- {
-  "sentence": "We have strong evidence today that studying a foreign language has a ripple effect, helping to improve student performance in other subjects.",
-  "author": "Richard Riley"
- },
- {
-  "sentence": "Speech is power: speech is to persuade, to convert, to compel. It is to bring another out of his bad sense into your good sense.",
-  "author": "Ralph Waldo Emerson"
- },
- {
-  "sentence": "One who speaks only one language is one person, but one who speaks two languages is two people.",
-  "author": "Turkish Proverb"
- },
- {
-   "sentence": "Language is very powerful. Language does not just describe reality. Language creates the reality it describes.",
-   "author": "Desmond Tutu"
- },
- {
-   "sentence": "It is astonishing how much enjoyment one can get out of a language that one understands imperfectly.",
-   "author": "Basil Lanneau Gildersleeve"
- },
- {
-   "sentence": "Speak a new language so that the world will be a new world.",
-   "author": "Rumi"
- },
- {
-   "sentence": "Every language is a world. Without translation, we would inhabit parishes bordering on silence.",
-   "author": "Francis George Steiner"
- },
- {
-  "sentence": "We should learn languages because language is the only thing worth knowing even poorly.",
-  "author": "Kato Lomb"
-}
-]
+[{
```

Comment:
Hey @tiresc, thanks for contributing! Just have one quick comment.

Hope you can fix the formatting of this file using a formatter such as [this](https://jsonformatter.curiousconcept.com/) to lessen the number of commits (seems like the whole file was changed when only a few lines were added) 😅. Use **2 Space Tab** for the JSON template.

Reply:
Ok, I hopefully have fixed the formatting, I for some reason the file got messed up when I copied it from the JS linter.

Reply:
Thanks for fixing but it seems like it’s still detecting some changes that are not supposed to be added :( What quotes were you trying to add? Maybe I can help out with the formatting if it helps!

Reply:
Oh no, I'm not sure what I've changed to make it detect that. 
I wanted to add just one quote by Yeats.
"sentence": "Think like a wise man but communicate in the language of the people.",
"author": "William Butler Yeats"

Reply:
Fixed the spacing issue! :)